### DISCUSSION TITLE
Doesn't he have a way to display emoji?

### DISCUSSION THREAD

**Discussion by Arbureva** [AllenDang/giu#561] [Q&A] [answered]
![emoji](https://user-images.githubusercontent.com/96331179/194215976-9bc0d8b7-07ad-446b-a374-e04ac84f7c25.png)
look at this

**Comment by Arbureva** [comment]
😋🤓😇️😀 this is my testing emoji

**Comment by AllenDang** [comment, marked as answer]
@Locter9001 You need to add a font which supports emoji.

**Reply by Arbureva** [reply]
Now he only shows emojis, not other characters, such as Chinese
```Golang
EmojFontInfo := g.AddFont("/lib/seguiemj.ttf", 14)
g.Style().SetFont(model.EmojFontInfo).To(
	g.Selectable(model.AllMsgs[i].OriginalMsg),
),
```

**Reply by gucio321** [reply]
Maybe use AddDefaultFont

**Reply by Arbureva** [reply]
@gucio321 I used `giu.setDefaultFont()`, im not found `addDefaultFont`

**Reply by Arbureva** [reply]
but still no use

**Comment by Arbureva** [comment]
@AllenDang @gucio321 emm, I just used `g.SetDefaultFont()` to set the emoji font and found that things are better, although my default font disappeared but it doesn't affect the display. But why can't I make two fonts coexist? Very strange phenomenon

**Reply by gucio321** [reply]
When using style.setfont the only font used is emoji font while the SetDefaultFont adds font to fonts list and all of them are used

**Reply by Arbureva** [reply]
I got it. thank you very much
